### ISSUE TITLE
Incorrect assessment of "call misses a required argument" for variadic function

### ISSUE THREAD

**Issue by pschultz** [scrutinizer-ci/php-analyzer#577]
Consider this code fragment:

```php
function f(StdClass ...$args) 
{
}

f();
```

The analyzer incorrectly reports that the call to f is missing required arguments.
